### PR TITLE
Update roadmap and todo list, add todo's labels

### LINKED ISSUE #124: Update Roadmap
Update Roadmap in line with https://docs.google.com/document/d/1RMZw_LTr2RBQbmCoQQ8V8eWMysAa8zJi0-UFGIBrNzk/edit?usp=sharing

Let me know if you need anything else.

When completed on the website, I'll update in docs.alephium.org.

### PR DESCRIPTION
**Pull request by mvaivre**
Fixes #124

You can see the changes here: www2.alephium.org

IMO, it's a lot of new items, making the roadmap and todo list really long.
Maybe we should filter a little bit and focus on essential content? cc/ @MaudSimon

### CODE REVIEW

File: src/content/homepage.md [hunk 1]
```diff
@@ -203,30 +203,84 @@ milestonesSection:
...
+      - order: 0
+        text: Launch Browser Wallet Extension
+        when: 'March 2023'
+      - order: 1
+        text: Leman Network Upgrade on Mainnet
+        when: '27.03.2023'
+        isMajor: true
+      - order: 2
+        text: Cross-chain interoperability
+        when: '03 2023'
```
Comment (nop33) [inline src/content/homepage.md]: Should be changed to `Mar 2023` to match the format we used in the past.

File: src/content/homepage.md [hunk 2]
```diff
@@ -203,30 +203,84 @@ milestonesSection:
...
+        text: NFT prototype alpha released
+        when: 'Q4 2022'
+      - order: 13
+        text: Improved DEX prototype
+        when: 'Q4 2022'
+    - year: 2023
+      entries:
+      - order: 0
+        text: Launch Browser Wallet Extension
+        when: 'March 2023'
```
Comment (nop33) [inline src/content/homepage.md]: Should be changed to `Mar 2023` to match the format we used in the past.

**Comment by nop33** [comment]
> IMO, it's a lot of new items, making the roadmap and todo list really long.
Maybe we should filter a little bit and focus on essential content?

I totally agree. It's way too long. Our website starts having the length of a scam landing page 😆 

Maybe we could group certain entries together to summarize them?

**Comment by mvaivre** [comment]
> Maybe we could group certain entries together to summarize them?

Exactly. I would do that.

File: src/content/homepage.md [hunk 3]
```diff
@@ -203,30 +203,84 @@ milestonesSection:
...
         isMajor: true
+      - order: 8
+        text: Improve documentation of the core infrastructure
+        when: 'H2 2022'
+      - order: 9
+        text: Improve the Alephium SDK with more features
+        when: 'H2 2022'
+      - order: 10
+        text: Improve the robustness and efficiency of the full node &  explorer backend
+        when: 'H2 2022'
```
Comment (mvaivre) [inline src/content/homepage.md]: I would merge those into one, smth along those lines: `Enhance the core platform's robustness and efficiency while improving documentation.`

File: src/content/homepage.md [hunk 4]
```diff
@@ -203,30 +203,84 @@ milestonesSection:
...
+      - order: 11
+        text: Alpha version of multi-guardians bridge on the Testnet
+        when: 'Nov 2022'
+      - order: 12
+        text: NFT prototype alpha released
+        when: 'Q4 2022'
+      - order: 13
+        text: Improved DEX prototype
```
Comment (mvaivre) [inline src/content/homepage.md]: Should we show the DEX launch date instead?

File: src/content/homepage.md [hunk 5]
```diff
@@ -203,30 +203,84 @@ milestonesSection:
...
+        when: 'Mar 2023'
+      - order: 8
+        text:  Improved node APIs and SDK
+        when: 'Mar 2023'
+      - order: 9
+        text:  Improved difficulty adjustment algorithm (DAA)
+        when: 'Mar 2023'
+      - order: 10
+        text:  Improved contract storage for security and performance
+        when: 'Mar 2023'
```
Comment (mvaivre) [inline src/content/homepage.md]: Those 9 items are all attached to March 2023. Maybe there's a way to merge some of them?

File: src/content/homepage.md [hunk 6]
```diff
@@ -260,53 +314,99 @@ milestonesSection:
...
-        - text: Mobile wallet
+        - text: 'Desktop Wallet v2.0'
+          label: Feature
           complete: false
+        - text: 'Mobile wallet (Android and iOS)'
+          label: Launch
+        - text: 'Alephium’s Bridge Launch on Testnet'
+          label: Launch
+        - text: 'Alephium’s Bridge Launch on Mainnet'
+          label: Launch
```
Comment (mvaivre) [inline src/content/homepage.md:302]: For the todos, maybe it's enough to just keep one item: `Alephium’s Bridge Launch`? It's kind of implicit that we deploy to testnet before mainnet IMO.

**Comment by MaudSimon** [comment]
Please let me know if you need me to work more on the items.

**Pull request merged**